recipe/meta.yaml

```diff
@@ -90,3 +90,4 @@ extra:
     - rth
     - adrinjalali
     - glemaitre
+    - jeremiedbb
```

Comment:
FWIW only this change is the only one that needs to be done manually. Re-rendering will pick up the rest 🙂

Reply:
thanks @jakirkham , good to know :)